components/homme/src/share/cxx/GllFvRemap.cpp

```diff
@@ -16,7 +16,7 @@
 namespace Homme {
 
 void init_gllfvremap_c (int nelemd, int np, int nf, int nf_max,
-                        bool theta_hydrostatic_mode,
```

Comment:
I'm assuming all these changes are b/c there's a problem getting bool values in a device lambda on SYCL, or something like that?

If that's the case, I would keep bool everywhere, and simply convert to int in the C++ structures that do need an int on SYCL. That would allow to leave lots of other files (including the F90 files) unchanged.